cmd/feed-ingress/main.go

```diff
@@ -102,3 +106,42 @@ func readFile(path string) []byte {
 	}
 	return data
 }
+
+func configureHealthPort(controller ingress.Controller) {
+	http.HandleFunc("/health", checkHealth(controller))
+
+	go func() {
+		log.Error(http.ListenAndServe("localhost:"+strconv.Itoa(healthPort), nil))
```

Comment:
Ok I changed it to listen on all interfaces. Also changed the port since 12001 is already used in `/etc/services` and it was showing up as some IBM service port.